artifacts/AdditionalSetup.ps1

```diff
@@ -115,6 +115,64 @@ Add-Content $artifactSettings -Value ('$TenantId         = "' + "$TenantId" + '"
 Add-Content $artifactSettings -Value ('$SyncMode         = "' + "$SyncMode" + '"')
 Add-Content $artifactSettings -Value ('$Scope            = "' + "$Scope" + '"')
 
+if ([string]::IsNullOrEmpty($env:enablePerformanceCounter) -or $($env:enablePerformanceCounter).ToLower() -eq "true") {
```

Comment:
@PatrickGrabensteiner are you trying to enable by default? Wouldn't it make more sense to do something like if (env -ne "false")? Or am I missing your goal?

Reply:
@tfenster You're right, i simplified the condition